helm/observability-operator/files/alertmanager/alertmanager.yaml.helm-template

```diff
@@ -23,12 +23,11 @@ route:
   - receiver: heartbeat
     matchers:
     - alertname="Heartbeat"
-    # TODO(@team-atlas): This label is needed for now because we support legacy and new heartbeat alerts. Lets remove it when we remove https://github.com/giantswarm/prometheus-rules/pull/1094/files#diff-2cc8f3328e242b6c86769bfb52b286c7146e074dd244ebb76a0025c838f33d0fR21
-    - type="mimir-heartbeat"
```

Comment:
I guess we should remove this matcher selector before we remove the label from here?
https://github.com/giantswarm/prometheus-rules/blob/main/helm/prometheus-rules/templates/platform/atlas/alerting-rules/mimir.rules.yml#L22

I'm not sure I understand why we needed both label matchers though.

Reply:
Oh true

Reply:
https://github.com/giantswarm/prometheus-rules/pull/1617